src/common/session/session-entity.js

```diff
@@ -31,7 +32,8 @@ const model = {
 export const SESSION_EVENTS = {
   PAUSE: 'session-pause',
   RESET: 'session-reset',
-  RESUME: 'session-resume'
+  RESUME: 'session-resume',
+  CROSS_TAB_UPDATE: 'session-cross-tab-update'
```

Comment:
Can this just be called `UPDATE`

Reply:
`SESSION_EVENTS.UPDATE` / `session-update` should only ever happen when the NRBA session got updated, as a subset of storage updates

Reply:
If we do this we should re-arrange some of the code.  Theres some places that are setting the mode directly,  If we tap into the session entity update, we should just have that be the one place that gets set, and then the handler updates the SR state (i think).  Theres some chicken vs egg stuff happening by tapping into a generic update event if we dont restructure some stuff

Reply:
I think we might benefit from adding a "type" or something to the emitted event so we can tell the difference between same-page session updates and cross-tab updates